packages/manager/apps/pci-public-ip/src/pages/order/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './OrderPage';
```

Comment:
We agreed last time not to use indexes for pages/components exports ( i have no problème with that, just question of following the same pattern everywhere :) ) 
https://stash.ovh.net/projects/UXMANAGERS/repos/documentation/pull-requests/23/diff#docs/guides/manager/development-guidelines/application-folder-structure.md